docs/pages/Configuration.md

```diff
@@ -96,25 +96,144 @@ However, it is recommended that modules include their namespace in the virtual p
 resources:
   configuration:
     paths:
-      "/resources/my/library/module-xy/": lib
-      "/resources/my/library/module-xy-min/": dist
+      /resources/my/library/module-xy/: lib
+      /resources/my/library/module-xy-min/: dist
 ````
 
 ### Encoding of `*.properties` files
-*Available since UI5 CLI [`v1.7.0`](https://github.com/SAP/ui5-cli/releases/tag/v1.7.0)*
 
-By default, the UI5 Tooling expects `*.properties` files to be `ISO-8859-1` encoded. 
+!!! info
+    This configuration is available since UI5 CLI [`v1.7.0`](https://github.com/SAP/ui5-cli/releases/tag/v1.7.0)
 
-If your project uses `UTF-8` encoding for these files (for example because it has been created in SAP WebIDE), you need to set the `propertiesFileSourceEncoding` configuration property.
+By default the UI5 Tooling expects different encodings for `*.properties` i18n files, depending on a projects specification version:
```

Comment:
... depending on the project's specification version:

Reply:
Done